ee/modules/610-service-with-healthchecks/crds/doc-ru-servicewithhealthchecks.yaml

```
@@ -271,55 +266,57 @@ spec:
-                    Должно быть либо `ClientIP`, либо `None`.
-                    
-                    По умолчанию — `None`.
+                    Должно быть указано значение либо `ClientIP`, либо `None`.
+
                     [Подробнее.](https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies)
                 sessionAffinityConfig:
-                  description: sessionAffinityConfig содержит конфигурации привязки сессий.
+                  description: Содержит конфигурации привязки сессий.
                   properties:
                     clientIP:
-                      description: clientIP содержит конфигурации привязки сессий на основе IP-адреса клиента.
+                      description: Содержит конфигурации привязки сессий на основе IP-адреса клиента.
                       properties:
                         timeoutSeconds:
                           description: |-
                             `timeoutSeconds` указывает время удержания сессии для типа `ClientIP` в секундах. 
                             
-                            Значение должно быть >0 && <=86400 (для 1 дня), `если ServiceAffinity` == `ClientIP`. 
-                            
-                            Значение по умолчанию — 10800 (для 3 часов).
+                            Значение должно быть >0 && <=86400 (для 1 дня), `если ServiceAffinity` == `ClientIP`.
                 trafficDistribution:
                   description: |-
-                    `TrafficDistribution` предоставляет возможность выразить предпочтения относительно того,
-                    как трафик распределяется между конечными точками сервиса. Реализации могут использовать это поле как подсказку, но не обязаны строго ей следовать.
+                    `TrafficDistribution` позволяет задать предпочтения по распределению трафика между конечными точками сервиса. Реализации могут учитывать это поле как рекомендацию, но не обязаны строго его соблюдать.
                     
                     Если поле не задано, реализация применяет стратегию маршрутизации по умолчанию. Если установлено значение `PreferClose`, реализации должны приоритизировать конечные точки, которые топологически близки (например, в той же зоне).
                 type:
                   description: |-
-                    `type` определяет, как сервис будет доступен. По умолчанию используется `ClusterIP`. Допустимые варианты: ExternalName, ClusterIP, NodePort и LoadBalancer. 
-                    ClusterIP выделяет внутренний IP-адрес для балансировки нагрузки между конечными точками. Конечные точки определяются селектором или, если он не указан, вручную с помощью объектов `Endpoints` или `EndpointSlice`.
+                    `type` определяет доступность сервиса.
+                    
+                    По умолчанию используется `ClusterIP`. Допустимые варианты: ExternalName, ClusterIP, NodePort и LoadBalancer. 
```

```suggestion
                    По умолчанию используется `ClusterIP`. Допустимые варианты: ExternalName, ClusterIP, NodePort и LoadBalancer. 
```
@mortelumina98 Давай отрендерим и глянем - там из спеки подтянется массив допустимых значений скорее всего и такие случаи не нужно дублировать.